lib/js/belt_Id.js

```diff
@@ -2,12 +2,48 @@
 
 var Curry = require("./curry.js");
 
+function MakeComparableU(M) {
+  var cmp = M[/* cmp */0];
+  return /* module */[/* cmp */cmp];
+}
+
+function MakeComparable(M) {
+  var cmp = function (a, b) {
+    return Curry._2(M[/* cmp */0], a, b);
+  };
+  return /* module */[/* cmp */cmp];
+}
+
 function comparableU(cmp) {
   return /* module */[/* cmp */cmp];
```

Comment:
it seems `comparableU` is slightly more efficient, it saves one allocation of a temporary module, but it should not matter in  practice, since it is a one time effort